recipes/sentry-crashpad/all/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            test_env_dir = "test_env"
+            tools.mkdir(test_env_dir)
+            bin_path = os.path.join("bin", "test_package")
+            handler_exe = "crashpad_handler.exe" if self.settings.os == "Windows" else "crashpad_handler"
+            handler_bin_path = os.path.join(self.deps_cpp_info["sentry-crashpad"].rootpath, "bin", handler_exe)
```

Comment:
```suggestion
            handler_exe = "crashpad_handler.exe" if self.settings.os == "Windows" else "crashpad_handler"
            handler_bin_path = os.path.join(self.deps_cpp_info["sentry-crashpad"].rootpath, "bin", handler_exe)
```

You don't need to pass the absolute path. By run_environment, it's already included in PATH, and with priority order, so even if there is another app with same name, that one provided by Conan should be used first.

Reply:
The absolute path is passed as an argument.
When applying the following patch:
```patch
diff --git a/recipes/sentry-crashpad/all/test_package/conanfile.py b/recipes/sentry-crashpad/all/test_package/conanfile.py
index 724a4675..1b6e6555 100644
--- a/recipes/sentry-crashpad/all/test_package/conanfile.py
+++ b/recipes/sentry-crashpad/all/test_package/conanfile.py
@@ -16,6 +16,5 @@ class TestPackageConan(ConanFile):
             test_env_dir = "test_env"
             tools.mkdir(test_env_dir)
             bin_path = os.path.join("bin", "test_package")
-            handler_exe = "crashpad_handler.exe" if self.settings.os == "Windows" else "crashpad_handler"
-            handler_bin_path = os.path.join(self.deps_cpp_info["sentry-crashpad"].rootpath, "bin", handler_exe)
-            self.run("%s %s/db %s" % (bin_path, test_env_dir, handler_bin_path), run_environment=True)
+            handler_exe = "crashpad_handler.exe" if self.settings.os == "Windows" else "crashpad_handler"e)
+            self.run("%s %s/db %s" % (bin_path, test_env_dir, handler_exe), run_environment=True)
```
the following error occurs:
```
----Running------
> bin/test_package test_env/db crashpad_handler
-----------------
[23258:23258:20210527,173156.407616:FATAL double_fork_and_exec.cc:131] execv crashpad_handler: No such file or directory (2)
```